gradle.properties

```diff
@@ -8,21 +8,28 @@
 #     http://www.apache.org/licenses/LICENSE-2.0
 #
 ### dependencies
+# For Nautilus 0-13-w11
+pravegaFlinkConnectorVersion=0.6.0
+pravegaCredentialsVersion=0.6.0
+pravegaVersion=0.6.0
+
+
 bucket4jVersion=4.4.1
 commonsCLIVersion=1.4
 flinkScalaVersion=2.12
 flinkVersion=1.7.2
+flinkShortVersion=1.7
 junitVersion=4.12
 logbackVersion=1.2.3
 logstashLogbackEncoderVersion=4.11
 nettyBoringSSLVersion=2.0.17.Final
 nettyVersion=4.1.30.Final
 # Below is for Nautilus 0.12.0
-pravegaCredentialsVersion=0.6.0-2345.298015f-0.12.0-W5-001.4e5c9a1
+# pravegaCredentialsVersion=0.6.0-2345.298015f-0.12.0-W5-001.4e5c9a1
```

Comment:
Go ahead and remove this line and other obsolete lines.

Reply:
done , please review